tools/runners/run-test-suite.sh

```diff
@@ -168,9 +173,11 @@ do
         status_code=$?
     fi
 
+    testfile=$(echo $cmd_line | sed s/" "/"\n"/g | tail -1 | sed s/"\/"/"\n"/g | tail -1)
```

Comment:
What is this trying to achieve? Is this some kind of a reinvented basename?

Reply:
The first sed command is to extract only the filename from the `$cmd_line`. The second sed command is not needed, but I didn't know about basename.